src/libraries/sendtohelix.proj

```diff
@@ -40,6 +40,11 @@
     <DotNetCliVersion>$([System.Text.RegularExpressions.Regex]::Match($(GlobalJsonContent), '(%3F&lt;="dotnet": ").*(%3F=")'))</DotNetCliVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.20308.1</MicrosoftDotNetXHarnessCLIVersion>
```

Comment:
Use the property in Versions.props instead?

Reply:
There was none, but I added it.